specification/azuredata/resource-manager/readme.md

```diff
@@ -0,0 +1,79 @@
+# azuredata
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for azuredata.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the azuredata.
+
+```yaml
+openapi-type: arm
+tag: package-2017-03-01-preview
+```
+
+### Tag: package-2017-03-01-preview
+
+These settings apply only when `--tag=package-2017-03-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2017-03-01-preview'
+input-file:
+  - Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-ruby
```

Comment:
add one more line:
`- repo: azure-sdk-for-net`

Reply:
Done